### DISCUSSION TITLE
Error create administrator account during installation

### DISCUSSION THREAD

**Discussion by Wojo1982** [Security-Onion-Solutions/securityonion#12529] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

other (please provide detail below)

### CPU

2

### RAM

16GB

### Storage for /

1TB

### Storage for /nsm

512GB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello. I try install Security Onion 2.4.50 airgap version but when I write eamil address for create administrator account for web intereface I still have info "write valid email". I try a few version email but nothink to work. 
Please help me. 

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
Are you trying to create a username with punctuation in it? That may not be supported.

**Reply by jertel** [reply]
Email addresses need to be specified in lower case, as well.

**Reply by Wojo1982** [reply]
Hi thanks for help. I try all and I found bug, I change DNS search domain. When I leave it at default searchdomain.local then everythink work. 
Best regards.

**Comment by Wojo1982** [comment]
Changed data in DNS in search domain option fixed all.
